src/test/components/TrackContextMenu.test.js

```diff
@@ -128,6 +177,10 @@ describe('timeline/TrackContextMenu', function() {
       trackItem.simulate('click');
       expect(getHiddenGlobalTracks(getState()).has(trackIndex)).toBe(false);
     });
+
+    xit('can present a disabled isolate item on non-process tracks', function() {
+      // TODO - We should wait until we have some real tracks without a thread index.
```

Comment:
We don't have the data to express this yet, so I would like to defer until after we add some tracks without a thread index.